streams/src/main/java/org/apache/kafka/streams/processor/internals/ClientUtils.java

```diff
@@ -52,12 +51,6 @@
 public class ClientUtils {
     private static final Logger LOG = LoggerFactory.getLogger(ClientUtils.class);
 
-    public static final class QuietStreamsConfig extends StreamsConfig {
```

Comment:
The new `InternalStreamsConfig` is a "quiet config" and thus we don't need this helper any longer